CMakeLists.txt

```diff
@@ -1,6 +1,15 @@
 cmake_minimum_required( VERSION 3.0 FATAL_ERROR )
 set( CMAKE_VERBOSE_MAKEFILE ON )
 
+# Configure the Android toolchain before the project start
+if( CINDER_TARGET )
```

Comment:
The compiler checks seem to happen when you ```project()``` is called. I think on most platform this is fine, but on Android since we have to do a manual toolchain setup, it needs to happen before ```project()```. Certainly not married to this solution, so if we can move stuff around to make it more sane, would be great.